### DISCUSSION TITLE
Selecting multiple resources

### DISCUSSION THREAD

**Discussion by DeHades666** [aquasarus/Starfield-Interactive-Smart-Slate#6] [Ideas]
I honestly feel like an idiot (stronger words are applicable and more apt)  asking for someone to do something.. but, here we go.. 
Is there a way or could there be a way to select multiple resources to be searched for on planets?
 For example, say I'm running low on Beryllium, looking to set up an outpost to mine it. I search for which planets have it, but wait, I need some Neodymium and Europium too..  I know I can check the 3 individually and see if a planet or moon appears in all 3 searches, but if I could select all 3 resources at once and voila Eridani IV-b could be somewhere to place an outpost. Just a crude example, no idea if it could be implemented or not, not an app breaker if not, just a thought.
 Can't say that I'm great at explaining either, so hope you caught some of the idea =s

**Comment by aquasarus** [comment]
Good explanation! This is actually a feature I initially wanted to add. It was cut from my v1 scope because I wanted to focus on cataloging lifeform data. I'm working on outpost tracking right now, and will look into adding this afterward :). In the meantime, Frederik's [spreadsheet](https://www.reddit.com/r/Starfield/comments/16g54cy/starfield_complete_list_of_resources_for_every/) has this, plus a lot more outpost planning features!

**Reply by DeHades666** [reply]
Fair enough, the lifeform data is a godsend (as mentioned in another post) and the outpost tracking will also be extremely beneficial. I can see the benefit of Frederik's spreadsheet, however, I'd prefer to at least scan a planet in game before knowing it has this and this resource etc, getting to the system as and when the quests allow.  Sebkul in the first comment says "You are doing Gods work my friend. Everyone wants to see this, nobody wants to spend the time to do it." - I'll do the time =) It'll take me quite a bit longer though! So be it. =)
